source/core/ExecutionContext.cpp

```diff
@@ -450,6 +450,24 @@ Int32 /*ExecSlicesResult*/ ExecutionContext::ExecuteSlices(Int32 numSlices, Int3
     return reply;
 }
 //------------------------------------------------------------
+void ExecutionContext::ExecuteTillNextStopPoint()
```

Comment:
"ExecuteAllClumpsTillNextDebugPoint" might be a more accurate name.

Reply:
Done